CHANGES.rst

```diff
@@ -45,6 +45,10 @@
 - Adding support for the VODataService 1.2 nrows attribute on table
   elements [#503]
 
+- registry.search now introspects the TAP service's capabilities and
+  only offers extended functionality or optimisations if the required
+  features are present [#386]
```

Comment:
As this is user facing, I would think the information that now it's possible to use other registries is important to list, or to list as well. 

That being said, I would go ahead and merge this as is so I can go ahead and do more tests and cleanups, but please do advise a rephrasing and I'll fix up the changelog before the release.